docker-compose.prod.yml

```diff
@@ -1,37 +1,44 @@
 version: '3.7'
 
 services:
-  web:
+  web_prod:
     build:
       context: ./app
-      dockerfile: Dockerfile.prod
-    command: gunicorn app.wsgi:application --bind 0.0.0.0:8000
+      dockerfile: Dockerfile
+    command: ["--production"]
     volumes:
-      - static_volume:/home/app/web/app/static
-      - media_volume:/home/app/web/app/media
+      - static_volume_prod:/home/app/web/app/static
+      - media_volume_prod:/home/app/web/app/media
     expose:
       - 8000
     env_file:
-      - ./.env.prod
+      - ./.env
+    environment:
+      DEBUG: 0
+      DJANGO_SETTINGS_MODULE: app.settings.production
     depends_on:
       - db
```

Comment:
Should this "depends_on" just be removed? It doesn't seem necessary, as it is isn't present in the Dev version, and it's currently breaking the build. Otherwise will need to change to `db_prod`.

Reply:
ah, excellent point. Dropped that line in 330eec0